cms/djangoapps/contentstore/management/commands/tests/test_export.py

```diff
@@ -0,0 +1,58 @@
+import unittest
+import ddt
+from django.core.management import CommandError
+from tempfile import mkdtemp
+
+from xmodule.modulestore.tests.factories import CourseFactory
+from contentstore.management.commands.export import Command
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from xmodule.modulestore.django import modulestore
+
+
+class TestArgParsingCourseExport(unittest.TestCase):
+    """
+    Tests for parsing arguments for the `export` management command
+    """
+    def setUp(self):
+        super(TestArgParsingCourseExport, self).setUp()
+
+        self.command = Command()
+
+    def test_no_args(self):
+        errstring = "export requires two arguments: <course id> <output path>"
+        with self.assertRaisesRegexp(CommandError, errstring):
+            self.command.handle()
+
+
+@ddt.ddt
+class TestCourseExport(ModuleStoreTestCase):
+    """
+    Test exporting a course
+    """
+    def setUp(self):
+        super(TestCourseExport, self).setUp()
+
+        # Make directory at the current location
+        self.temp_dir = mkdtemp(dir="")
```

Comment:
Do you also want to test that this works with directories that end and don't end with `/`?